src/watts/plugin.py

```diff
@@ -198,3 +193,51 @@ def prerun(self, params: Parameters, filename: Optional[str] = None):
         self.render_template(params_copy, filename=filename)
         for render_template in self.extra_render_templates:
             render_template(params_copy)
+
+    def postrun(self, params: Parameters, name: str, **kwargs) -> Results:
+        """Read simulation results and create results object
+
+        Parameters
+        ----------
+        params
+            Parameters used to generate input files
+        name
+            Name of the plugin
+        **kwargs
+            Keyword arguments for Results subclasses
+
+        Returns
+        -------
+        Results object
+        """
+
+        # Determine time, inputs and outputs
+        time = datetime.fromtimestamp(self._run_time * 1e-9)
+        inputs = [p.name for p in self.extra_inputs]
+        inputs.append(self.input_name)
```

Comment:
This makes the main input file in the middle of `inputs` instead of in the beginning. Thus, it is not compatible with 
https://github.com/paulromano/watts/blob/6e3b6736a0801293a57293fe8bbe705fe3c8c592/src/watts/plugin_moose.py#L53-L54, which assumes the main input file the the first one.

Reply:
@nstauff @paulromano MOOSE multiapps's csv output cannot be read correctly due to this.

Reply:
Ah, nice catch! I just pushed a commit that should fix this.

Reply:
It works now. I will submit a PR to update MultiStep_Griffin-BISON-Sockeye_MR input files once this is merged.